felix/fv/bpf_map_resize_test.go

```diff
@@ -84,6 +88,35 @@ var _ = infrastructure.DatastoreDescribe("_BPF-SAFE_ Felix bpf test configurable
 		}
 	}
 
+	It("should copy data from old map to new map", func() {
```

Comment:
It feels like some UTs in bpf package might be more light-way easier to control tests for this. That would allow you to more easily test the content (also if expired values are being removed along the way), all the corner cases, error cases etc.